documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -66,6 +69,23 @@ spec:
 |`controller_block_overhead`
 |Fixed amount of additional space allocated in persistent block volumes. This setting is applicable for any `storageclass` that is block-based. It can be used when data, such as encryption headers, is written to the persistent volumes in addition to the content of the virtual disk. Note, this setting can only be changed using the CLI.
 |`0`
+
+
+|`vsphere_osmap_configmap_name`
+|Configuration map for vSphere source providers. This configuration map maps the incoming operation system of the incoming VM to a {virt} preference name. This configuration map needs to be in the namespace where `forklift` is deployed, that is, where `forklift-controller` is running.
```

Comment:
```suggestion
|Configuration map for vSphere source providers. This configuration map maps the incoming operating system of the incoming VM to a {virt} preference name. This configuration map needs to be in the namespace where `forklift` is deployed, that is, where `forklift-controller` is running.
```
???

Reply:
@anarnold97 Based on one of Liran's comments -- I tagged you after the comment so that you can see the context. Meanwhile, I changed "operation" to "operating"